### ISSUE TITLE
Attempting to install 2.4.0 stuck on empty dependency

### ISSUE THREAD

**Issue by zhartmeh** [Esri/arcgis-python-api#2142]
**conda version : 24.9.1**

Per documentation, using the following command:
`conda install -c esri arcgis arcgis-mapping`
 
![EmptyDep](https://github.com/user-attachments/assets/9b15c6f8-5f33-4de0-a0f5-124f99f1a410)

And it just stalls there. 

I tried reverting the solver to classic but this failed as well. See Attached.
![Solver](https://github.com/user-attachments/assets/ae766007-4271-4abd-9d51-1f1339af4275)

I reverted the solver back and attempted to install with no dependencies and still same error. See Attached.
![NoDeps](https://github.com/user-attachments/assets/010d0e25-07a5-4137-b707-197527571603)

**Comment by zhartmeh** [comment]
**Update:**
After letting the command prompt sit for >1hour, the following errors are returned. Hope this helps!?

![image](https://github.com/user-attachments/assets/7d98b31e-faa3-4e78-8f09-984199fd188e)

**Comment by achapkowski** [comment]
are you using something that requires the latest version of requests?

**Comment by zhartmeh** [comment]
Do you mean the arcgis Python API (2.4.0). The new features are[ summarized here](https://developers.arcgis.com/python/latest/guide/overview24/). There have been numerous changes related to the (former) arcgis.mapping module.

**Comment by nanaeaubry** [comment]
@zhartmeh `requests` is a package
It seems you are using the latest version and it is causing conflicts. I would attempt installing in a fresh environment

**Comment by zhartmeh** [comment]
Yep this is a fresh environment. Thanks for the suggestion though.

**Comment by nanaeaubry** [comment]
Closing this issue since we cannot help further, please contact support if you need help installing
